ui/app/templates/components/selectable-card-container-graph.hbs

```diff
@@ -0,0 +1,50 @@
+<SelectableCard
+  @class="grid-item-http"
```

Comment:
if this is a `SelectableCard` component, i think this should also have the `selectable-card` class name. this makes it easier to notice what kind of component it is when we're inspecting elements in the DOM, and it's good for testing purposes since it means we could assert that X number of elements with this class name exists. 

implementing the default class name i mentioned earlier is an option that'll help!